src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/SecurityServiceDispatcher.cs

```diff
@@ -278,7 +278,7 @@ private async Task<IServiceChannelDispatcher> GetInnerChannelDispatcherAsync(ICh
              {
                  throw DiagnosticUtility.ExceptionUtility.ThrowHelperError(new NotSupportedException(SR.GetString(SR.UnsupportedChannelInterfaceType, typeof(TChannel))));
              }*/
-
+           // await ((ICommunicationObject)securityChannelDispatcher).OpenAsync();
             return securityChannelDispatcher;
```

Comment:
Should this be commented out?

Reply:
nope.. I removed the line to avoid confusion